CDMS_VCS/avg_cloudiness.py

```diff
@@ -0,0 +1,35 @@
+import vcs, cdms2, sys, os
+
+file = cdms2.open(os.path.join(vcs.sample_data, "clt.nc"))
+clt = file("clt", time=('1985-1', '1986-1'))
+t = clt.getTime()[::3]
+canvas = vcs.init()
+alt_c = None
+move_y = 0
+for i in range(len(t)-1):
+
+    mean = (clt[i] + clt[i+1] + clt[i+2]) / 3
```

Comment:
So this isn't working quite the way you think...

If you add this code:
```
times = i, i + 1, i + 2
import cdtime
for time in times:
    reltime = cdtime.reltime(clt.getTime()[time], clt.getTime().units)
    print reltime.tocomponent()
```

you'll see that you're actually getting the months

i | i + 1 | i + 2
---|-------|------
Jan | Feb | March
Feb | March | April
March | April | June
April | June | July

If you think about it, this makes sense; `range(len(t) - 1)` should be equal to `[0, 1, 2, 3]`, which means you're just adding 1 or 2 to those values. What you actually want is every third value in the range of 0 to 12; you should be able to just generate a range using the same basic idea as the slice syntax you used above.